AdobeStockImageAdminUi/view/adminhtml/layout/cms_wysiwyg_images_index.xml

```diff
@@ -8,7 +8,7 @@
 <layout xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/layout_generic.xsd">
     <referenceContainer name="root">
         <block class="Magento\Backend\Block\Template" template="Magento_AdobeStockImageAdminUi::panel.phtml" name="stock.panel">
-            <uiComponent name="adobe_stock_images_listing"/>
+            <uiComponent name="adobe_stock_images_listing" aclResource="Magento_AdobeStockImageAdminUi::save_preview_images" />
```

Comment:
I think we can enforce ACL in the panel block (parent layout element)

Reply:
done